namui/src/namui/system/typeface/load_sans_typeface_of_all_languages.rs

```diff
@@ -18,6 +20,12 @@ async fn load_fallback_font_typefaces() -> Result<(), Box<dyn std::error::Error>
     let typeface = get_noto_color_emoji_typeface().await?;
```

Comment:
Here is typeface you need

Reply:
Good idea

Reply:
I thought that there will be many typefaces and iter them with `get_fallback_font_typefaces`.
I think I thought too much about the future. I'll follow yours.

Reply:
You didn't followed. Check code again please.

Reply:
nvm